pages/index.vue

```diff
@@ -18,13 +18,6 @@
       </template>
     </HeroBase>
 
-    <SlideBase
-      v-for="slide in slides"
-      v-bind="slide"
-      :key="slide.slug"
-      class="h-screen"
-    />
-
     <VimeoVideo :video="futureStaticVideo">
```

Comment:
The name of the component should be SectionVideo, because we have SectionBase and we don't want to tie our codebase to a specific video provider (we might switch to YouTube or another, in the future); try to keep related functionalities near each other by following the (n|N)ounAdjectiveVerb naming convention.